{"name": "登录异常，测试数据是:{'accounts': 'yaoyao', 'pwd': '123456', 'exp': '密码错误'}", "status": "passed", "parameters": [{"name": "args", "value": "{'accounts': 'yaoyao', 'pwd': '123456', 'exp': '密码错误'}"}], "start": 1599053412420, "stop": 1599053412548, "uuid": "01952d10-becf-4011-b875-ca09ca1e2f00", "historyId": "9b1c39a42a17c4eb61b2e9ff0cabd0ab", "testCaseId": "acfd50c72596b11b4392c40d7652bccf", "fullName": "script.test_login.TestLogin#test_login_error", "labels": [{"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-60EEKMHU"}, {"name": "thread", "value": "14368-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test_login"}]}